streams/src/main/java/org/apache/kafka/streams/processor/internals/Tasks.java

```diff
@@ -304,4 +310,26 @@ void addTask(final Task task) {
         }
         allTasksPerId.put(task.id(), task);
     }
+
+    public void deprioritizeNamedTopology(String name) {
+        for (Task task: readOnlyTasks) {
+            if (task.id().namedTopology().equals(name)){
+                misbehavingTasks.add(task);
+            }
+        }
+        for (Task task: misbehavingTasks) {
```

Comment:
I think suspending will work. however I am not sure it will survive thread restarts. I may need to store the topology names that are in each state to repopulate each of these lists in new threads